bitpay/client.py

```diff
@@ -86,10 +86,23 @@ def verify_invoice_params(self, price, currency):
   def response_error(self, response):
     raise BitPayBitPayError('%(code)d: %(message)s' % {'code': response.status_code, 'message': response.json()['error']})
 
-  def unsigned_get_request(self, payload):
+  def unsigned_request(self, path, payload=None):
+    """
+    generic bitpay usigned wrapper
+    passing a payload will do a POST, otherwise a GET
```

Comment:
For clarity: we want to avoid having this kind of logic in a method: the user should know whether they are getting or posting and call the appropriate get/post method. This is why I'm closing this pull request, although I'll be happy to reopen it if changes are made.